frameworks/helloworld/tests/test_placement.py

```diff
@@ -239,7 +242,7 @@ def setup_constraint_switch():
     agents = shakedown.get_private_agents()
     some_agent = agents[0]
     other_agent = agents[1]
-    print("agents", some_agent, other_agent)
+    log.info('Agents: {} {}'.format(some_agent, other_agent))
```

Comment:
The python logger supports format string out the box (sort of). This line can be replaced with:
```python
log.info('Agents: %s %s', some_agent, other_agent)
```